utility/pigpio/compatibility.cpp

```diff
@@ -1,15 +1,12 @@
+#include <sys/time.h>
+#include <time.h>
```

Comment:
Is this the same header? Should we go with just time.h?

Reply:
I don't know, try it and see if it works!

Reply:
They are not the same.
- sys/time.h is linux-specific and resolves to `/usr/include/<system-arch>/time.h`
- time.h is the std lib (I think) and resolves to `/usr/include/time.h`

Going with sys/time.h since these functions rely on linux-specific functions (`clock_nanosleep()`). Removing time.h does not affect anything.

Reply:
It turns out `clock_nanosleep()` is actually defined in `/usr/include/time.h` which is from GLibC. It isn't easy to track down, but I think time.h also eventually `#include <sys/time.h>`, and we aren't using anything specifically defined by sys/time.h.

Reply:
using time.h